packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -72,6 +73,15 @@ export interface VideoGalleryStrings {
   localVideoCameraSwitcherLabel: string;
 }
 
+/**
+ * @public
+ */
+export type LocalTileMode =
+  | 'default'
+  | 'floatingLocalVideo'
+  | /* @conditional-compile-remove-from(stable) meeting/calling-composite <Local-Camera-Switcher> */ 'controlsLocalVideo'
```

Comment:
See my comment about scoping of this type.

`layout = 'controlsLocalVideo'` doesn't make much sense in the context of the video gallery (I read that as: the layout of the video gallery is such that it controls the local video!)

Should this be a separate prop `enableLocalCameraSwitcher?: boolean` that is only meaningful when `layout = 'floatingLocalVideo'`. In that case, we should throw an exception if it is set with `layout = 'default'`.

Reply:
oh we have that prop already. I didn't think to use it. will make this refactor